src/orgnr.js

```diff
@@ -17,7 +17,7 @@ function gen() {
 	if (contr === 11) contr = 0;
 
 	if (contr === 10)
-       return null;
+    return null;
```

Comment:
Looks like spaces snuck in instead of tab, should be tab here:
```suggestion
	return null;
```

Reply:
This is resolved in #13